src/main/java/org/jenkinsci/plugins/workflow/pipelinegraphanalysis/StatusAndTiming.java

```diff
@@ -172,10 +174,27 @@ public static GenericStatus computeChunkStatus(@Nonnull WorkflowRun run,
                         return (lastNode.getError() == null) ? GenericStatus.SUCCESS : GenericStatus.FAILURE;
                     }
                 }
+
+                if (lastNode instanceof StepStartNode) {
```

Comment:
Huh?? Is this not going to return something totally meaningless for block-scoped steps other than `node`?